tests/php/Centreon/Domain/PlatformTopology/PlatformTopologyServiceTest.php

```diff
@@ -48,6 +50,21 @@ class PlatformTopologyServiceTest extends TestCase
      */
     protected $platformTopologyRepository;
 
+    /**
+     * @var HttpClientInterface|null $httpClient
+     */
+    protected $httpClient;
+
+    /**
+     * @var PlatformInformationServiceInterface
+     */
+    private $platformInformation;
```

Comment:
```suggestion
    private $platformInformationService;
```
platformInformation is ambiguous, as its also the name of the entity PlatformInformation.